src/parser/core/modules/Invulnerability.tsx

```diff
@@ -110,7 +110,7 @@ const DEFAULT_ACTOR_FILTER = filter<Actor>().team(Team.FOE)
 
 export class Invulnerability extends Analyser {
 	static override handle = 'invulnerability'
-	static override debug = false
+	static override debug = true
```

Comment:
hey az did you intentionally leave debug on?

Reply:
Zodiark dammit...